src/hooks/useContract.ts

```diff
@@ -47,7 +47,8 @@ export function useV2MigratorContract(): Contract | null {
 }
 
 export function useMiniChefContract(): Contract | null {
-  return useContract(MINICHEF_ADDRESS, MiniChefV2.abi, true)
+  const { chainId } = useActiveWeb3React()
+  return useContract(MINICHEF_ADDRESS[chainId || ChainId.AVALANCHE], MiniChefV2.abi, true)
```

Comment:
what if we create a hook `useChainId`
something like this
```js
const useChainId = () => {
  const { chainId } = useActiveWeb3React()
  reutrn chainId || ChainId.AVALANCHE
}
```

so we dont need to repeat `chainId || ChainId.AVALANCHE` everytime

Reply:
Resolved in the updated PR.
I have put the new hook useChainId wherever it was possible